client/verta/tests/utils.py

```diff
@@ -14,6 +16,12 @@
 from hypothesis import strategies as st
 
 
+# Jenkins workers had been getting lost during artifact tests;
+# we added this env var to ensure files are written to disk
+# (and not, say, into RAM)
+TEMPDIR_ROOT = os.environ.get("TEMPDIR_ROOT")
```

Comment:
Did you add this to the configuration setup?

Reply:
@hmacdonald-verta This is actually moved here from https://github.com/VertaAI/modeldb/pull/2799/files#diff-8d105224a574535507b29e892eb2171f2096c788362f47da0c4d31cf3a72b3c0L335-L337; it's already existed, so nothing needs to be added to our cluster config setup (if I'm understanding right)!

Reply:
I see! Okay 👍🏼